Q/Qiskit/build_tarballs.jl

```diff
@@ -0,0 +1,57 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Qiskit"
+version = v"2.2.3"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Qiskit/qiskit.git", "8f33595f5b9e9c99b7aa81002655d13f48c8ac1b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+export PYTHONHOME=${prefix}
+cd qiskit
+mkdir -p target
```

Comment:
```suggestion
cd $WORKSPACE/srcdir/qiskit
mkdir -p target
export PYTHONHOME=${prefix}
```
Just a minor style nit - this better consolidates the various commands together.